src/Symfony/Component/Form/FormErrorIterator.php

```diff
@@ -213,13 +213,15 @@ public function hasChildren()
     }
 
     /**
-     * Alias of {@link current()}.
-     *
      * @return self
      */
     #[\ReturnTypeWillChange]
     public function getChildren()
     {
+        if (!$this->hasChildren()) {
+            trigger_deprecation('symfony/form', '5.4', 'Calling "%s()" if the current element is not iterable is deprecated, call "%s" to get the current element.', __METHOD__, self::class.'::current()');
```

Comment:
Can you please add the `throw` statement that will be used in 6.0 so that it only needs to be uncommented in the `6.0` branch?